rosMsgs/s_robot_status robotStatus
---
